pkg/controller/config/operator_config.go

```diff
@@ -0,0 +1,25 @@
+package config
```

Comment:
Shouldn't this be in `main`? CLI stuff doesn't seem to fit `controller`...

I think it's OK to pass Atlas URI to our reconcilers during construction; everything else is already only used from `main`.